acasclient/interactions.py

```diff
@@ -27,7 +27,8 @@
     'comprises': 'is comprised of',
     'analyzes': 'is analyzed by',
     'is input structure for': 'has input structure',
-    'owns': 'is owned by'
+    'owns': 'is owned by',
+    'is compatible with': 'is compatible with',
```

Comment:
I know I came up with this, but I hate this particular interaction because it is reversible...
TODO for me: think of a better, more specific, directional term and replace this one.